### DISCUSSION TITLE
Welcome to discussions?

### DISCUSSION THREAD

**Discussion by splewis** [splewis/csgo-practice-mode#170] [General]
I just enabled this, since it seemed interesting, and perhaps more lightweight that the issues section.

Feel free to post any general questions, thoughts about the plugin, etc.

Note that I'd prefer this to *not* become a general "sourcemod/CSGO server" support channel, so please try to stick to things actually **relevant to practicemode**.

**Comment by melvinsjostrand** [comment]
Hello, my school was thinking of adding 2 of the plugins. The scrim and Pracc plugin but we cant because they both start the plugin with .setup. Can u change it somehow so we can have both the plugins?
